tgui/packages/tgui/interfaces/AresInterface.js

```diff
@@ -1,3 +1,7 @@
+// -------------------------------------------------------------------- //
+// Please ensure when updating this menu, changes are reflected in AresAdmin.js
```

Comment:
Is there a reason these aren't just one file with the differences hidden when it's not an admin using it? This, in its current state, is kind of weird for maintainability and preferably future people who touch this wouldn't have to change things in multiple places like this.

Reply:
The reason it's not the same menu is because of the scope of differences. My end goal is to have it so all the data is stored in the same place rather than localised to the ARES Interface Console but at the moment I'm trying to get the functionality sorted first.

Merging both menus is theoretically possible, but at the moment is beyond my capability to appropriately manage.
I have a very rough end state in mind for the whole of Project ARES, so I have no intentions of abandoning my work on it so long as I remain around CM and retain my capability to do so.

Reply:
Just to be clear, it doesn't really matter if you're going to be around or not. This problem is really only going to compound on itself and they should be merged now rather than forcing someone to deal with this in the future.